packages/k-ramel/src/reaction.js

```diff
@@ -1,19 +1,17 @@
-import { isRegExp, isString, isFunction } from 'lodash'
-
 const isMatching = (action, store) => matcher => ( // test matching
   // to a string
   (
-    isString(matcher) &&
+    (typeof matcher === 'string' || matcher instanceof String) &&
     action.type === matcher
   )
   // to a function
   || (
-    isFunction(matcher) &&
+    (typeof matcher === 'function' || matcher instanceof Function) &&
     matcher(action, store)
   )
   // to a regexp
   || (
-    isRegExp(matcher) &&
+    matcher instanceof RegExp &&
```

Comment:
I am not sure about this

Reply:
I think it's ok !
![image](https://user-images.githubusercontent.com/26094222/42593559-10b8e12a-854d-11e8-89df-882ac69e7645.png)